EpicCactus94.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "@EpicCactus94"
+project: "Self Balancing Reaction Wheel Cube"
+---
+
+# Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances 
+like pushing. I was inspired to build this project from videos online of how they're used in satelites.

```

Comment:
Can you link an example of such a video?

Reply:
Sure. 
https://www.youtube.com/watch?v=mH8a_UaOFHY&ab_channel=Engineeringandarchitecture

Reply:
Great, could you link it in your project proposal?

Reply:
It's already in the .md file in this pull request